bindings/python/htcondor2/_collector.py

```diff
@@ -39,9 +40,9 @@ class Collector():
     # In version 1, there was a distinct DaemonLocation type (a named tuple)
     # that `pool` could also be, but that functionality was never documented.
     #
-    def __init__(self, pool : Optional[Union[str, classad.ClassAd, List[str]]] = None):
+    def __init__(self, pool : Optional[Union[str, classad.ClassAd, List[str], Tuple[str]]] = None):
         """
-        :param pool:  A ``host:port`` string, or a list of such strings,
+        :param pool:  A ``host:port`` string, or a list or tuple of such strings,
```

Comment:
```suggestion
        :param pool:  A ``host:port`` string, list or tuple of such strings,
```

Reply:
I generally feel that Oxford commas are mandatory where applicable in technical documentation.  However, this is not an Oxford comma: "a string of such strings" is not sensible.

It would apparently be clearer to instead say (note that the last comma is Oxfordian):

A ``host:port`` string specifying a collector, or a list or tuple of such strings, or a ClassAd...

Arguably, the current phrasing became wrong when I switched the "...such as might..." clause from being in parenthesis to being comma-separated, since it only applies to the last element in the list, the comma is part of the last list item, and the list separators should be semicolons.

Reply:
Actually,

"A `host:port` string specifying a collector, a list or tuple of strings, or a ClassAd..." would be better.

Reply:
On third thought,

"A `:host:port` string specify a collector, a list (or tuple) of such strings, or a ClassAd..." would be betterer.

Reply:
I like your third thought.